src/shared/components/layout/sidebar/Sidebar.scss

```diff
@@ -0,0 +1,5 @@
+@import 'src/styles/common';
+
+.wrapper {
```

Comment:
`.wrapper` -> `.sidebar`? Same as the component name?

Reply:
✅